object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,43 @@
+/* exported omit */
+
+/*
+- create new empty array
+- create new empty object
+- use for...in loop to go through each key in source
+- push each key in source to empty array
+- loop through each key in key array
+- if key isn't included in keys array -> add this property to new object
+- return new object
+
+old code block
+function omit(source, keys) {
+  var sourceKey = [];
+  var obj = {};
+  for (var key in source) {
+    sourceKey.push(key);
+  }
+  for (var i = 0; i < sourceKey.length; i++) {
+    if (!Array.prototype.includes.call(keys, sourceKey[i])) {
+      obj[sourceKey[i]] = source[sourceKey[i]];
+    }
+  }
+  return obj;
+}
+
+---Revision-----
+- remove sourceKey array
+- use for...in loop to go through each key in source
+- determine if key is included in keys array
+- if not push key and value to obj
+
+*/
+
+function omit(source, keys) {
+  var obj = {};
+  for (var key in source) {
+    if (!Array.prototype.includes.call(keys, key)) {
```

Comment:
This complicated way to call this method is unnecessary. `if(!keys.includes(key)){`